docs/README.md

```diff
@@ -22,17 +22,17 @@ Before generating your store's sitemap, you might want to adjust whether product
 
 3. Run `vtex install vtex.store-sitemap@2.x` to install the Sitemap app.
 
-   > ℹ️ If you are using `vtex.edition-store@5.x`, skip step 3, as the `vtex.store-sitemap@2.x` app is installed by default in this version. Check out our [Edition App documentation](https://developers.vtex.com/docs/guides/vtex-io-documentation-edition-app) to know more about its different versions.
+   > ℹ️ If you're using `vtex.edition-store@5.x`, skip step 3, as the `vtex.store-sitemap@2.x` app is installed by default in this version. Check out our [Edition App documentation](https://developers.vtex.com/docs/guides/vtex-io-documentation-edition-app) to know more about its different versions.
 
 4. Run `vtex install vtex.admin-graphql-ide@3.x` to install the GraphQL admin IDE.
    
 5. In your browser, access the Admin and go to **Store Setting > Storefront > GraphQL IDE**.
 
-   ![graphql-ide](https://cdn.jsdelivr.net/gh/vtex-apps/store-sitemap@main/docs/graphql-ide.png)
+   ![graphql-ide](https://github.com/user-attachments/assets/5f28d91d-662d-485c-a071-aabc3a65cb71)
 
 6. From the dropdown list, choose the `vtex.routes-bootstrap@0.x` app.
 
-7. If this is **not the first time** you are generating your store's sitemap or if your store's routes **did not suffer any changes** since the last time you generated your store's sitemap, go to step 8. Otherwise, run the following query:
+7. If this is **not the first time** you're generating your store's sitemap or if your store's routes **didn't suffer any changes** since the last time you generated your store's sitemap, go to step 8. Otherwise, run the following query:
```

Comment:
```suggestion
7. If this is not the first time you're generating your store's sitemap or if your store's routes didn't suffer any changes since the last time you generated your store's sitemap, go to step 8. Otherwise, run the following query:
```